docs/account_management/gitlab_user.md

```diff
@@ -1,31 +1,57 @@
-# UAB GitLab Registration Instructions
+# UAB GitLab Overview
+
+Welcome to UAB GitLab! This is a UAB-specific [GitLab](https://about.gitlab.com/). GitLab is similar to [GitHub](https://github.com/), but hosted here at UAB on secure servers. 
+
+### Researcher GitLab Use Cases
```

Comment:
Heading levels are strange, right? Looks like It goes from the page title  on line 5 (single #)to a subheading level (###) on line 5.

Reply:
Yeah should definitely be two #'s